src/client.cpp

```diff
@@ -217,7 +217,7 @@ const char *GetBackendName(void)
 
 const char *GetBackendVersion(void)
 {
-  static const char *strBackendVersion = "UNKNOWN";
+  static const char *strBackendVersion = enigma ? enigma->GetServerVersion() : "unknown";
```

Comment:
Shouldn't "unkown" be localisable?

Reply:
I actually have several of these in my next PR, there are quite a few. This is one of them.